.gitignore

```diff
@@ -316,4 +317,7 @@ playground/*
 tutorial/*
 
 .out/
-.log/
\ No newline at end of file
+.log/
+
+output_dag.png
+Snakefile
```

Comment:
Could the Snakefile be part of the output folder? Not git tracked indeed, but easily reachable, e.g. if the out folder is `./out` then at `.out/Snakefile`

Reply:
ah, but it is already, solved, `def serialize_workflow(self, benchmark: Benchmark, output_dir: str = os.getcwd()):` does this

Reply:
Yes, Serialization of the snakefile can be output anywhere, by default is just the work directory.